rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] These variables are not only accessible: They can equally be modified: `attr_accessor :date, :book, :person`

Reply:
Will do, thank you